### PR TITLE
Rc 1.3.25

### PR DESCRIPTION
**Pull request by semuadmin**
# PyGPSClient Pull Request Template

## Description

RELEASE CANDIDATE v1.3.25

ENHANCEMENTS:

1. UBX Configuration Load/Save/Record facility will now accept u-center *.txt configuration files (for Generation 9+ devices only) as well as binary *.ubx files. Thanks for @wdwalker in #66 for suggestion.

FIXES:

1. Fix error when updating NTRIP Client dialog status (e.g. with 'unauthorized' errors)

CHANGES:

1. Bandit security analysis added to VS Code and GHA workflows.
1. Update min pygnssutils version to 1.0.7
1. Internal updates to VSCode and GHA workflows.

Fixes #66

## Testing

Please test all changes, however trivial, against the supplied pytest suite `tests/test_*.py`. Please describe any test cases you have amended or added to this suite to maintain >= 99% code coverage.

- [x] *.txt config conversion tested against [ArduSimple F9P RTK config files](https://www.ardusimple.com/configuration-files/)

## Checklist:

- [x] I agree to abide by the code of conduct (see [CODE_OF_CONDUCT.md](https://github.com/semuconsulting/pyubx2/blob/master/CODE_OF_CONDUCT.md)).
- [x] My code follows the style guidelines of this project (see [CONTRIBUTING.MD](https://github.com/semuconsulting/pyubx2/blob/master/CONTRIBUTING.md)).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] (*if appropriate*) I have added test cases to the `tests/test_*.py` unittest suite to maintain >= 99% code coverage.
- [x] I have tested my code against the full `tests/test_*.py` unittest suite.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) my commits.
- [x] I understand and acknowledge that the code will be published under a BSD 3-Clause license.

### CODE REVIEW

File: src/pygpsclient/strings.py
Comment (semudev2) [inline src/pygpsclient/strings.py:1]: remove commented out lines

File: tests/test_static.py
Comment (semudev2) [inline tests/test_static.py:1]: may add test to validate widgets.py setup?

**Review by semudev2 — CHANGES_REQUESTED** [review]

File: tests/test_static.py
Comment (semuadmin) [reply tests/test_static.py:1]: testwidgetgrid added

File: src/pygpsclient/strings.py
Comment (semuadmin) [reply src/pygpsclient/strings.py:1]: done

**Review by semudev2 — APPROVED** [review]

**Pull request merged**